integration/conftest.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import pytest
+
+import py42.sdk as _sdk
+
+HOST_URL = "https://console.us.code42.com"
```

Comment:
If we make this a parameter in `run_integration.py`, we can alternately choose to run the tests elsewhere more easily.

Reply:
NA applicable anymore as the file is removed.